ovirt/resource_ovirt_vm.go

```diff
@@ -376,6 +382,71 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 	vmBuilder.Cpu(cpu)
 
+	if storage_domain, _ := blockDevice.([]interface{})[0].(map[string]interface{})["storage_domain"]; storage_domain != "" && templateIDOK {
+
+		// Get the reference to the service that manages the storage domains
+		sdsService := conn.SystemService().StorageDomainsService()
+
+		// Find the storage domain we want to be used for virtual machine disks
+		sdsResp, err := sdsService.List().Search("name=" + storage_domain.(string)).Send()
+		if err != nil {
+			return fmt.Errorf("Failed to search storage domains, reason: %v", err)
+		}
+
+		tds, err := getTemplateDiskAttachments(templateID.(string), meta)
```

Comment:
this should be `tda` not `tds`

Reply:
Hi @rgolangh 
Inside function resourceOvirtVMCreate variable for TemplateDiskAttachments object was named as `tds`
Mentioned code:
```
        if templateIDOK {
                tds, err := getTemplateDiskAttachments(templateID.(string), meta)
                if err != nil {
                        return err
                }
                if len(tds) > 0 && blockDeviceOk {
                        device := blockDevice.([]interface{})
                        // check if a disk id was passed to block_device, fail if so.
                        if diskID, _ := device[0].(map[string]interface{})["disk_id"].(string); diskID != "" {
                                return fmt.Errorf("template_id with disks attached is conflict with block_device")
                        }
                }
                if len(tds) == 0 && !blockDeviceOk {
                        return fmt.Errorf("template has no disks attached, so block_device must be assigned")
                }
        }
```

I've consider that it is not necessary to change the variable naming format.
Are you sure that I need to change name from `tds` to the `tda` for TemplateDiskAttachments inside function resourceOvirtVMCreate?